packages/editor-ui/src/components/Node.vue

```diff
@@ -31,6 +31,12 @@
 				</div>
 
 				<div class="node-trigger-tooltip__wrapper">
+					<n8n-tooltip placement="top" manual :value="showCustomTooltip" popper-class="node-trigger-tooltip__wrapper--item">
```

Comment:
this should be moved to a new component `CanvasAddButton` instead of it being part of the node